src/app/components/core/configuration-manager/view-config-item/view-config-item.component.ts

```diff
@@ -61,7 +61,9 @@ export class ViewConfigItemComponent implements OnInit, OnChanges {
 
   public difference(obj, bs) {
```

Comment:
Looks like we should have BaseConfigComponenent or some common module for such methods